wgsl/index.bs

```diff
@@ -6082,13 +6086,13 @@ This analysis can be run in linear time, by analyzing the call-graph bottom-up (
 ### Examples ### {#behaviors-examples}
 
 Here are some examples showing this analysis in action:
-<div class='example wgsl expect-error' heading='Trivially dead code is rejected'>
+<div class='example wgsl expect-error' heading='Trivially dead code is allowed'>
    <xmp highlight='rust'>
     fn simple() -> i32 {
       var a: i32;
       return 0;  // Behavior: {Return}
-      a = 1;     // Error: by the rule for sequences of statements, the previous statement should have a behavior that include "Next"
-      return 2;
+      a = 1;     // Valid, statically unreachable code. Behavior: {Return}
+      return 2;  // Valid, statically unreachable code. Behavior: {Return}
```

Comment:
Nitpick  The behaviour of the _whole function_ is {Return}. That doesn't come through so clearly in the proposed text.

Reply:
I agree, this line might be interpreted to mean that the behavior of "a = 1" is {Return}, whereas it is {Next}. What has behavior {Return} is the function body as a whole.
This is just an editorial nitpick, not a deep issue with the PR, which I otherwise am happy with.

Reply:
I've clarified these types of examples to specify the statements behaviour, but also the overall resulting behaviour.